extension/src/plots/webview/contract.ts

```diff
@@ -1,15 +1,7 @@
 import { VisualizationSpec } from 'react-vega'
 import { Color } from '../../experiments/model/status/colors'
 
-// It is easier to keep the numerical order than the alphabetical one
-/* eslint-disable sort-keys-fix/sort-keys-fix */
-export const PlotNumberOfItemsPerRow = {
-  ONE: 1,
-  TWO: 2,
-  THREE: 3,
-  FOUR: 4
-}
-/* eslint-enable sort-keys-fix/sort-keys-fix */
+export const DefaultNumberOfItemsPerRow = 2
```

Comment:
> [N] Should use screaming snake case. e.g `NUM_OF_COMMITS_TO_SHOW`

I got confused and was wondering about this because of other variables not using this format. Will change that.

Reply:
> [Q] Should this be calculated based on the screen size?

Good question. I could try this as a follow up. Currently the default vale is set in the extension, so I'd have to reset everything to undefined to then calculate it in the webview. I'd probably select the middle value by default.